modules/mapper-extras/src/main/java/org/elasticsearch/index/mapper/extras/RankFeatureFieldMapper.java

```diff
@@ -62,24 +64,37 @@ public static class Builder extends FieldMapper.Builder {
             true
         );
         private final Parameter<Map<String, String>> meta = Parameter.metaParam();
+        private final Parameter<Float> nullValue = new Parameter<>("null_value", false, () -> null,
+            (n, c, o) -> o == null ? null : ((Number)o).floatValue(), m -> ft(m).nullValue, XContentBuilder::field,
+            Objects::toString).acceptsNull();
```

Comment:
Should we use `objectToFloat` function instead of `((Number)o).floatValue()` as we should accept string values as well.

Reply:
`Builder` should have a method `nullValue` that sets the value for `nullValue`

Reply:
Both done.